proposals/csharp-7.2/readonly-ref.md

```diff
@@ -257,9 +257,9 @@ The life time of the argument temporaries matches the closest encompassing scope
 The formal life time of temporary variables is semantically significant in scenarios involving escape analysis of variables returned by reference.
 
 ## Metadata representation of `in` parameters.
-When `System.Runtime.CompilerServices.IsReadOnlyAttribute` is applied to a byref parameter, it means that the the parameter is an `in` parameter.
+When `System.Runtime.InteropServices.InAttribute` is applied to a byref parameter, it means that the the parameter is an `in` parameter.
```

Comment:
I think this is wrong. The Roslyn compiler places a `System.Runtime.CompilerServices.IsReadOnlyAttribute` custom attribute (and optionally a `System.Runtime.InteropServices.InAttribute`) for the required custom modifier.

Note that `ParameterInfo.GetCustomAttributes()` may report a custom attribute of type `System.Runtime.InteropServices.InAttribute`, but that is actually a fiction produced by Reflection.

Reply:
(Furthermore, it might be good to be more precise about what "is applied to" means: Applied as a custom attribute, or as a required custom modifier? But that probably needs to be discussed in the issue first.)

Reply:
agreed, will close this and submit a new request with just the other change

Reply:
Why not wait until we have an answer from the C# language team?

Reply:
I was reviewing the tests in Roslyn to determine this. In any case, branches and PRs are cheap and easy to dismiss.